pages/thankyou.js

```diff
@@ -79,7 +79,15 @@ export default function Confirmation(props) {
             ) : (
               ""
             )}
-            <ActionButton id="resend_email" text={t("resendEmail")} tertiary />
+            {referrer === "signup" ? (
+              <ActionButton
+                id="resend_email"
+                text={t("resendEmail")}
+                tertiary
+              />
+            ) : (
+              ""
+            )}
```

Comment:
I think the resend button can just be completely removed, it doesn't look like it's used anywhere.

Reply:
Ah yes you are right. I thought we used it when subscribing. Removing...